Services/UI/OJS.Services.Ui.Models/Contests/ContestProblemServiceModel.cs

```diff
@@ -24,7 +24,7 @@ public class ContestProblemServiceModel : IMapExplicitly
 
         public int OrderBy { get; set; }
 
-        public int MaximumPoints { get; set; }
+        public short MaximumPoints { get; set; }
```

Comment:
Wierd. Any reason for that?

Reply:
To match the property in the Problem class, which is also short.